### PR TITLE
Update plugin enable cli reference documentation

### PR DESCRIPTION
**Pull request by vdemeester**
Follow-up of https://github.com/docker/docker/issues/28595

/cc @thaJeztah @tiborvass @anusha-ragunathan 

:frog: 

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: docs/reference/commandline/plugin_enable.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ Usage:  docker plugin enable PLUGIN
 Enable a plugin
 
 Options:
-      --help   Print usage
+      --help          Print usage
+      --timeout int   HTTP client timeout (in seconds)
```
Comment (thaJeztah) [inline docs/reference/commandline/plugin_enable.md]: Just discussed with @vdemeester that this should probably be a `duration`, to be consistent with other options.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by vdemeester** [comment]
author LGTM :sun_with_face:

File: docs/reference/commandline/plugin_enable.md (hunk 1 above)
Comment (anusha-ragunathan) [inline docs/reference/commandline/plugin_enable.md]: Duration is not necessary here. This was already discussed at https://github.com/docker/docker/pull/28595#discussion_r88989260

Comment (thaJeztah) [inline docs/reference/commandline/plugin_enable.md]: There's more options that don't need sub-second precision, perhaps we should include it in a UX review (I do understand your point though)

**Pull request merged**